src/screens/MealList/mealCard/MealCard.tsx

```diff
@@ -11,6 +11,7 @@ class MealCard extends React.Component<Props, State> {
 
     render() {
         return (
+            <View style={styles.container}>
```

Comment:
Hmm you shouldn't have to have done anything with this file

Reply:
Ah ok, sorry about that. Obviously feel free to leave